dts/arm/nordic/nrf52810.dtsi

```diff
@@ -5,6 +5,15 @@
 #include <dt-bindings/gpio/gpio.h>
 #include "nrf5_common.dtsi"
 
+#include <nordic/override.h>
+/*
+ * This value can be overridden at the board level or in an application specific
+ * dts/nordic/override.h file.
+ */
+#ifndef DT_DEFAULT_IRQ_PRIORITY
+#define DT_DEFAULT_IRQ_PRIORITY 1
```

Comment:
Use the lowest priority as default, which is 6 IIRC for nRF52, as highest priority is used for Zero Latency IRQs.

Similarly for rest of the nRF52, nRF53 and nRF91 series